doc/org/build/gameplay.md

```diff
@@ -0,0 +1,722 @@
+      destroyAtExtremeDistance
+      floatyObject={{
+        flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE,
+        releaseGravity: 0
+      }}
+      networkedFloatyObject={{
+        flags: FLOATY_OBJECT_FLAGS.MODIFY_GRAVITY_ON_RELEASE
+      }}
+      rigidbody={{ collisionGroup: COLLISION_LAYERS.INTERACTABLES, collisionMask: COLLISION_LAYERS.HANDS }}
+      physicsShape={{ halfExtents: [0.22, 0.14, 0.1] }} /* TODO Physics shapes*/
+      scale={[1, 1, 1]}
+    />
+  );
+}
+```
+
+Although `EntityDef` s are written with `jsx` syntax, this is not `React`. The `jsx` syntax allows us to describe our desired scene graph, entities, and components. The definition is static, and there should be no expectation of &ldquo;re-rendering&rdquo; (as in `React` or `three-fiber`). Semantically, an `EntityDef` is equivalent to a model file with component data. `EntityDef` s are meant to be easy to edit by hand and to version control.
+
+For `network instantiated` entities, `template` functions are grouped together with `permission` information to form a named `Prefab`. More information about `network instantiated` entities can be found in the networking documentation.
+
+
+<a id="org156c909"></a>
+
+## Creating model files
+
+Equivalently, hubs components can be added to nodes in a GLTF file with the `MOZ_hubs_components` extension.
+
+The `hubs-blender-exporter` is a Blender add-on that helps artists do this.
+
+Spoke also includes component data in the gltf files that it exports and uploads.
+
+
+<a id="org2319bae"></a>
+
+## Entity creation is synchronous
+
+It is important to realize that `renderAsEntity` is a synchronous function. That is, it will immediately return a valid `EntityID` with a corresponding `Object3D` added to the scene graph.
+
+The presence of some components (like `MediaLoader`) cause systems to begin asynchronous work. In the case of `MediaLoader`, this work can include downloading model or image files, loading them with the GLTF loader, and ultimately creating additional entities and components. But the entity at the root of this `Object3D` hierarchy will be created synchronously/immediately when `renderAsEntity` runs.
+
+
+<a id="org628683b"></a>
+
+## Inflation
+
+
+<a id="org20a38aa"></a>
+
+### What does `renderAsEntity` do?
+
+In the example above, we show a `template` function (`MediaPrefab`) that takes some `InitialData` (the `MediaLoaderParams`) and returns an `EntityDef`. We then said that the `EntityDef` can be passed to `renderAsEntity` which will (synchronously) add entities and components to the world. We also said this was equivalent to loading a model file (GLTF).
+
+Question: How does `renderAsEntity` (and whatever loads models) accomplish this? Answer: By running `inflators`.
+
+
+<a id="org357d40d"></a>
+
+### `Inflator` s
+
+An `inflator` is a function that transforms a <span class="underline">description</span> of some components/entities into real components and real entities.
```

Comment:
This might be a bit confusing. If I'm right a single inflator itself initializes and adds some real components to a real entity, not entities (plural).